demos/custom-tabs-oauth/src/main/java/com/google/androidbrowserhelper/demos/customtabsoauth/MainActivity.java

```diff
@@ -17,13 +17,69 @@
 package com.google.androidbrowserhelper.demos.customtabsoauth;
 
 import android.app.Activity;
+import android.content.Intent;
+import android.net.Uri;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ProgressBar;
+import android.widget.TextView;
 
 public class MainActivity extends Activity {
+    private static final String AUTHORIZATION_ENDPOINT = "https://github.com/login/oauth/authorize";
+    private static final String CLIENT_ID = "<github-client-id>";
+    private static final String CLIENT_SECRET = "<github-client-secret>";
+    private static final String REDIRECT_URI =
+            "https://oauth-custom-tabs.glitch.me/oauth/auth-callback.html";
+
+    private static final OAuthManager OAUTH_MANAGER =
+            new OAuthManager(CLIENT_ID, CLIENT_SECRET, AUTHORIZATION_ENDPOINT, REDIRECT_URI);
+
+    private Button mLoginButton;
+    private TextView mUserText;
+    private ProgressBar mProgressBar;
+    private boolean loggedIn = false;
```

Comment:
You don't need the `= false`, that's the default value for booleans. (Also, `mLoggedIn`.)

Reply:
done (and `mLoggedIn`).